src/backend/plaf/utils.rs

```diff
@@ -0,0 +1,65 @@
+use polyexen::plaf::Plaf;
+
+// adapted from polyexen-demo
+// keep a version for Chiquito plaf backend that we can maintain independently
+pub fn alias_replace(plaf: &mut Plaf) {
```

Comment:
remove it

Reply:
Done. Was previously used for MiMC7 example boilerplate. The chunk has been deleted.